### PR TITLE
Geomarketing

### PR DESCRIPTION
**Pull request by Robinlovelace**
Making this PR so it builds on travis as a test - no need to merge!

### CODE REVIEW

**Comment by Robinlovelace** [comment]
I'm still not able to reproduce compilation of the chapter - getting this error message on the latest version of this branch:

```
Quitting from lines 177-231 (08-location.Rmd) 
Error in summarise_impl(.data, dots) : 
  Evaluation error: invalid 'type' (closure) of argument.
Calls: local ... NextMethod -> summarise.tbl_df -> summarise_impl -> .Call
Execution halted
Error in Rscript_render(f, render_args, render_meta) : 
  Failed to compile 08-location.Rmd
Calls: <Anonymous> ... render_new_session -> tryCatch -> tryCatchList -> Rscript_render
Execution halted
```

File: 08-location.Rmd [hunk 1]
```diff
@@ -96,7 +96,7 @@ input = mutate_all(input, funs(ifelse(. %in% c(-1, -9), NA, .)))
 After the data preprocessing, we convert the table into a raster stack.
 First, we use the x- and y-coordinates to convert the `data.frame` into a `SpatialPolygonsDataFrame` (we do not use sf-objects here, since the **raster** package does not support `sf`).
 <!-- make sure that this is the case -->
-`gridded()` converts `SpatialPolygonsDataFrame` into a `SpatialGridDataFrame` which in turn can be used as an input for **raster**'s `stack()` function.
+`gridded()` converts a `SpatialPolygonsDataFrame` into a `SpatialGridDataFrame` which in turn can be used as an input for **raster**'s `stack()` function.
```
Comment (Robinlovelace) [inline 08-location.Rmd]: Note for later: we should use `sf` objects when this becomes possible.

Comment (jannes-m) [reply 08-location.Rmd]: Sure if **raster** ever gets updated...

File: 08-location.Rmd [hunk 2]
```diff
@@ -225,7 +225,13 @@ st_centroid(metros) %>%
 -->
 
 However, visual inspection reveals eight metropolitan areas whereas the dissolving comes up with nine.
-This is because one polygon just touches the corner of another polygon (western Germany, Cologne/Düsseldorf area).
+This is because one polygon just touches the corner of another polygon (western Germany, Cologne/Düsseldorf area; see Fig. \@ref(fig:metro-areas)).
+
+```{r metro-areas, echo = FALSE, fig.width = 1, fig.height = 1, fig.cap = "The aggregated raster (resolution: 20 km) with the identified metropolitan areas (golden rectangles) and the corresponding names."}
+knitr::include_graphics("figures/08_metro_areas.png")
```
Comment (Robinlovelace) [inline 08-location.Rmd]: This looks like a great plot. Enjoying seeing this chapter come together!

**Comment by jannes-m** [comment]
ok, will delete then this figures in the future. Thanks for the advice!

**Pull request closed without merging**